### PR TITLE
Resolve #8 . Added introduction and section about chess. Also added other relevant .tex files for future edits

### LINKED ISSUE #8: Chess as a Game

### PR DESCRIPTION
**Pull request by MadsHeilmann**

### CODE REVIEW

File: src/whitelist.txt
Comment (audio-engineer) [inline src/whitelist.txt:1]: @MadsHeilmann File lacks final newline. Please format files before committing and pushing

**Review by audio-engineer — CHANGES_REQUESTED** [review]

File: src/whitelist.txt
Comment (MadsHeilmann) [reply src/whitelist.txt:1]: Screenshot 2024-03-01 140043

I pressed CTRL + ALT + L on all the files. In the screenshot is the message i get. I dont understand why it is necessary to add newline when the formater does not do it and when both actions do not care for it.

Comment (audio-engineer) [reply src/whitelist.txt:1]: @MadsHeilmann https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3. We already use EditorConfig to insert a final newline on save, not on format. Which is why `Ctrl + Alt + L` didn't make a difference.

File: src/codestyling.tex
Comment (audio-engineer) [inline src/codestyling.tex:1]: @MadsHeilmann Why did you add this if you didn't add any code listings? And why configure the language to C if there will most likely be no C code listings in the report?

**Review by audio-engineer — CHANGES_REQUESTED** [review]
@MadsHeilmann I am not happy with how there are two chapters called `Problem Analysis` now. It's not necessary. It would be much better to add your new content to the existing chapter, replacing the notes, while leaving the other notes as they are.

File: src/whitelist.txt
Comment (MadsHeilmann) [reply src/whitelist.txt:1]: Okay - i see. That is enabled now.

**Comment by MadsHeilmann** [comment]
> @MadsHeilmann I am not happy with how there are two chapters called `Problem Analysis` now. It's not necessary. It would be much better to add your new content to the existing chapter, replacing the notes, while leaving the other notes as they are.

I dont think i understand exactly. As I see it, there is only one chapter for the problem analysis in problem-analysis.tex

File: src/codestyling.tex
Comment (MadsHeilmann) [reply src/codestyling.tex:1]: I was thinking that we were going to have to add it eventually anyway, so might as well add it now. For now it is just a copy of the file we had in P1. I can delete the contents of the codestyling.tex file if it is necessary

File: src/appendix/appendix.tex [hunk 1]
```diff
@@ -0,0 +1,13 @@
+% textidote: ignore begin
+\chapter*{Appendices}
+
+\section*{Appendix A}\label{sec:appendix-b}
```
Comment (Danar435) [inline src/appendix/appendix.tex]: Both appendix sections are mislabeled

File: src/introduction/introduction.tex [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+% textidote: ignore end
+
+Learning is something we do every day.
+It is something most people enjoy and strive to experience.
+However, learning can also be difficult.
+In this report we will explore the concept of collaborated assisted learning and how software can assist in this method.
+We will be focusing our analysis around the age-old game of chess.
+
+AI and chess engine popularity has risen greatly in the past years and have been developed to a point that they
+outperform humans.
```
Comment (Danar435) [inline src/introduction/introduction.tex:12]: Source here would be nice

**Review by Danar435 — CHANGES_REQUESTED** [review]

File: src/appendix/appendix.tex (hunk 1 above)
Comment (MadsHeilmann) [reply src/appendix/appendix.tex]: Woops. Will be fixed

File: src/introduction/introduction.tex (hunk 2 above)
Comment (MadsHeilmann) [reply src/introduction/introduction.tex:12]: It is just the introduction, why it does not require sources I don't think. Furthermore the point will be sourced in the problem analysis later, why I don't think it is necessary to include sources in the introduction

**Comment by audio-engineer** [comment]
@MadsHeilmann 
image

**Comment by MadsHeilmann** [comment]
> @MadsHeilmann 
> 
> image

Interesting. It did not look like that on my local compilation. I will fix it :)

**Comment by audio-engineer** [comment]
@MadsHeilmann Please wait with merging. I am trying to fix some issues with compilation etc.

File: src/main.tex [hunk 3]
```diff
@@ -24,11 +26,18 @@
...
 
+    \input{introduction/introduction}
     \input{problem-analysis/problem-analysis}
+    \input{problem-statement/problem-statement}
 
+    % textidote: ignore begin
+    \addcontentsline{toc}{chapter}{Bibliography}
     \bibliographystyle{IEEEtran}
-    \bibliography{main,problem-analysis/multiplayer}
+    \bibliography{problem-analysis/chess,problem-analysis/multiplayer}
```
Comment (audio-engineer) [inline src/main.tex]: @MadsHeilmann You will have to change this to use `\addbibresource` instead

**Review by audio-engineer — CHANGES_REQUESTED** [review]

**Review by audio-engineer — APPROVED** [review]

**Review by Danar435 — APPROVED** [review]
